manifests/run/via_cron/device.pp

```diff
@@ -44,9 +45,15 @@
     # The above, via the interval_to_cron_time function:
     # $cron_time = device_manager::interval_to_cron_time($run_interval, fqdn_rand(max(1,min($run_interval, 59)), $name))
 
+    if ($run_user == '') {
+      $optional_user = ''
+    } else {
+      $optional_user = " --user=${run_user}"
+    }
+
     cron { "run puppet device target ${name}":
       ensure  => $cron_ensure,
-      command => "${device_manager::run::command} ${device_manager::run::arguments} --target=${name}",
+      command => "${device_manager::run::command} ${device_manager::run::arguments} --target=${name}${optional_user}",
       user    => $::identity['user'],
```

Comment:
What is the benefit of `--user` over changing this here?

Reply:
Do you mean setting `user => $run_user` instead of `--user=${run_user}` ... or both?

Reply:
I was expecting a change that sets `user => $run_user`, so now I'm confused why you chose this route instead.

Reply:
From this comment in #46 

I'd like folks to be able to deploy puppet device configurations from a root-running agent into a unprivileged user. There are so many `user` contexts here! You just let me know what you want ...

Reply:
do you want to schedule a call next week so we can resolve this confusion?

Reply:
I'll say yes to any rationale to get together with you!